ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -695,6 +699,221 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- condensing_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- condensing_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- oil_heater_status:
+  - ON
+  - OFF
+- inhibit_start_time_sensor
+- load_limit_capacity
+- total_compressor_count
+- compressor_run_time_sensor
+- guidevane_percentage_command
+- oil_temperature_sensor
+- oil_pump_differential_pressure_sensor
+- exhaust_gas_compressor_temperature_sensor
+- bearing_temperature_sensor
+- winding_temperature_sensor
+- compressor_temperature_setpoint
+- compressor_temperature_sensor
+- load_compressor_percentage_setpoint
+- compressor_power_sensor
+- refrigerant_valve_percentage_sensor
+- run_time_sensor
+- standby_compressor_alarm:
+  - ACTIVE
+  - INACTIVE
+- high_compressor_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_compressor_pressure_alarm:
+  - ACTIVE
+  - INACTIVE
+- compressor_current_alarm:
+  - ACTIVE
+  - INACTIVE
+- compressor_run_time_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_high_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_overflow_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_low_level_alarm:
+  - ACTIVE
+  - INACTIVE
+- fan_speed_frequency_setpoint
+- compressor_speed_frequency_setpoint
+- start_mode:
+  - ON
+  - OFF
+- reset_alarm:
+  - ACTIVE
+  - INACTIVE
+- reset_command:
+  - ON
+  - OFF
+- chilled_water_isolation_valve_mode:
+  - AUTO
+  - MANUAL
+- pump_speed_frequency_setpoint
+- chilled_water_isolation_valve_percentage_setpoint
+- chilled_water_flowrate_alarm:
+  - ACTIVE
+  - INACTIVE
+- end_alarm:
+  - ACTIVE
+  - INACTIVE
+- start_alarm:
+  - ACTIVE
+  - INACTIVE
+- simulation_command:
+  - ON
+  - OFF
+- condensing_supply_water_isolation_simulation_command:
+  - ON
+  - OFF
+- stop_command:
+  - ON
+  - OFF
+- closed_command:
```

Comment:
im not sure what the point of this is; what is closed? and why would ON/OFF be the states?

Reply:
The command to close isolation valve provided it is open

Reply:
this, `open_command`, and `stop_command` seem very odd. is there no valve command that you tell to both open and close? why are there two separate outputs to do effectively the same function in reverse?  And why does `stop_command` exist at all?

if this must be modeled, we would need abstract types that contain these two fields so that a user would know what is meant here, something like: 
```
TPVC:
  description: "Two position valve control with individual control points for opening and closing the valve"
  uses:
  - open_command
  - close_command
```

Reply:
The open, stop and close command points has been removed